library/gdalframework/src/main/java/it/geosolutions/imageio/gdalframework/GDALUtilities.java

```diff
@@ -610,8 +610,58 @@ public static boolean isGDALAvailable() {
         return available;
     }
 
+    /**
+     * This will ensure that java has loaded the GDAL library (<code>gdalalljni</code> or <code>gdaljni</code>).
+     * 
+     * <ul>
+     * <li>GDAL 3+ -- GDAL.jar will automatically load the library (nothing for us to do)</li>
+     * <li>GDAL 2.3+ -- load "gdalalljni" lib</li>
+     * <li>GDAL <2.3 -- load "gdaljni" lib</li>
+     * </ul>
+     * <p>
+     * Use <code>java -XshowSettings:properties<code> to check <code>java.library.path</code>.
+     * On linux, you will probably need to set <code>LD_LIBRARY_PATH</code>.
+     */
+    private static void ensureGDALLibraryLoaded() {
+        // Called from synchronized block in loadGDAL
+        try {
+            gdal.AllRegister(); //will throw if library not already present
+            if (gdal.GetDriverCount() > 0){
+                //at least one driver was recognized
+                LOGGER.log(Level.FINE,"ensureGDALLibraryLoaded: library is already loaded.");

```

Comment:
Why "already"? Makes it seem like it got loaded twice or something like that.

Reply:
Is it not accurate? the library is already loaded ...

Reply:
Seems like an error message of the sorts of "why are you loading again the library when it was loaded already"?
In fact, it just got loaded for the first time, no?

Reply:
I will change to the more accurate: `ensureGDALLibraryLoaded: Confirmed library is loaded and available.`